src/transform.js

```diff
@@ -29,3 +29,37 @@ module.exports = async function(source, options) {
 };
 
 module.exports.version = babel.version;
+
+function injectCaller(opts) {
+  if (!supportsCallerOption()) return opts;
+
+  return Object.assign({}, opts, {
+    caller: Object.assign(
+      {
+        name: "babel-loader",
+        supportsStaticESM: true,
+      },
+      opts.caller,
+    ),
+  });
+}
+
+let supportsCallerOptionFlag = undefined;
+function supportsCallerOption() {
```

Comment:
maybe add a comment that this can be removed after a while?

Reply:
Can do.